Resources/config/data-collector.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="httplug.message_journal" class="Http\HttplugBundle\Collector\MessageJournal" public="false">
+            <tag name="data_collector" template="HttplugBundle::webprofiler.html.twig" priority="200"
+                 id="httplug"/>
+        </service>
+
+        <service id="httplug.plugin.history" class="Http\Client\Plugin\HistoryPlugin" public="false">
```

Comment:
lets use a more specific service name here to prevent confusion with generally useable plugins.